src/legacy/core_plugins/kibana/public/visualize/np_ready/editor/editor.js

```diff
@@ -643,9 +647,9 @@ function VisualizeAppController(
   }
 
   $scope.unlink = function() {
-    if (!$state.linked) return;
+    if (!$scope.linked) return;
 
-    $state.linked = false;
+    $scope.linked = false;
```

Comment:
When testing the link/unlink saved search did not work when using browser back/forward buttons - probably because the `linked` state is saved outside of the state? Would it make sense to also put it in app state? Looks like it is currently part of it.

Reply:
Good catch! Updated!